src/challenges/challengeOne/5.Modal/Modal.test.tsx

```diff
@@ -1,8 +1,54 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Modal } from "./Modal";
-
-// describe("Modal", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Modal, ModalProps } from "./Modal";
+
+describe("Modal", () => {
+  const onClose = jest.fn();
+  const onConfirm = jest.fn();
+
+  const renderComponent = (additionalProps?: Partial<ModalProps>) => {
+    const defaultProps: ModalProps = {
+      isOpen: true,
+      onClose,
+      onConfirm,
+      children: <div>Modal Content</div>,
+      headerText: "Modal Header",
+      showActions: true,
+    };
+
+    render(<Modal {...defaultProps} {...additionalProps} />);
+  };
+
+  it("renders without error", () => {
```

Comment:
In our 'renders without error' we want to test that the screen is not null because it really tests there are no errors in the component that is preventing it from rendering. 

We want a separate test for the header text, which is what is being tested inside this test.

Reply:
Yes, I'll check that part.